build2/KoreBuild.ps1

```diff
@@ -0,0 +1,100 @@
+Write-Host -ForegroundColor Green "Starting KoreBuild 2.0 ..."
+
+$RID = "win7-x64";
+if($env:PROCESSOR_ARCHITECTURE -eq "x86") {
+    $RID = "win7-x86";
+}
+
+$ToolsDirectory = $env:KOREBUILD_TOOLS_DIRECTORY
+if(!$ToolsDirectory) {
+    $ToolsDirectory = Join-Path $env:LOCALAPPDATA "Microsoft\AspNetCoreBuildTools\$RID"
```

Comment:
This might not be the best solution even if configurable because then you can have only one version of the build system and if you build a different repo you might update the build scripts for everyone. Also, doing multiple builds on the same machine at the same time might crash